### PR TITLE
VEGA-204 - enabling visit requests for other programs

### PR DESCRIPTION
**Pull request by spullen**
https://thirdchannel.atlassian.net/browse/VEGA-204

* Updates to the create/update form (mainly around products, and some copy changes)
* Updating scheduling related views (some still in progress)

Also see PRs for ThirdChannel, Oddjob, and Procrastination

### CODE REVIEW

File: js/apps/procrastination/views/schedule/upcoming/details_modal.js [hunk 1]
```diff
@@ -2,18 +2,24 @@ define(function (require) {
...
+
     DetailsModalView = Backbone.View.extend({
         template: HandlebarsTemplates['procrastination/schedule/upcoming/visit_label'],
+
         className: 'details-modal',
-        render: function () {
-            this.model.isVega = this.model.jobDetails.programUUID == '219f1927-7997-4c03-a5c9-4a6b4d792d3e';
+
+        render: function() {
+            console.log('here', this.model);
```
Comment (eliottwiener-thirdchannel) [inline js/apps/procrastination/views/schedule/upcoming/details_modal.js]: debug log left in?

File: js/apps/procrastination/views/schedule/upcoming/details_modal.js [hunk 2]
```diff
@@ -2,18 +2,24 @@ define(function (require) {
...
+            this.model.retailerReferenceSheet = context.retailerReferenceSheet;
+            this.model.isJobRequest = this.model.jobDetails.jobRequestUuid !== undefined || this.model.jobDetails.jobRequestUuid !== null;
```
Comment (eliottwiener-thirdchannel) [inline js/apps/procrastination/views/schedule/upcoming/details_modal.js]: should this be `&&`?

Comment (spullen) [inline js/apps/procrastination/views/schedule/upcoming/details_modal.js]: yup, good catch

File: js/apps/procrastination/views/schedule/upcoming/details_modal.js (hunk 1 above)
Comment (spullen) [inline js/apps/procrastination/views/schedule/upcoming/details_modal.js]: removed

**Pull request merged**